docs/release-notes/8.1.asciidoc

```diff
@@ -47,6 +47,7 @@
 ==== Known issues
 * An {endpoint-sec} integration bug prevents benign Windows files from being deleted under certain circumstances.
 * On macOS versions before 12.4, if {elastic-endpoint} is used with other products that monitor or manage network traffic (such as antivirus programs, firewalls, or VPNs), users might experience network connection issues. To resolve this issue, upgrade to macOS 12.4 or later.
+* Indicator match rules will encounter execution errors if they use an index that was generated by a value list ({pull}133457[#133457]).
```

Comment:
Not sure if we want to be more specific here and mention it would specifically be the `.items-*` index? @nkhristinin

Reply:
@yctercero @nkhristinin another possible revision:

Avoid configuring indicator match rules to use indices populated from value lists. Indicator match rules cannot use the `.lists` system index and will encounter execution errors when ran. As a workaround, you can....<insert workaround>. 


Also, I can insert this ^ as a note [here](https://www.elastic.co/guide/en/security/current/rules-ui-create.html#indicator-value-lists) to make users aware of the limitations and workaround.

Reply:
This sounds good to me! Minus the workaround that we determined won't be included. Thanks so much!